stubs/pyinstaller/PyInstaller/utils/win32/winmanifest.pyi

```diff
@@ -1,127 +0,0 @@
-from _typeshed import Incomplete, ReadableBuffer, Unused
```

Comment:
Seems like this file still exists, why was it deleted? The contents do look very different from what's currently in the stub.

Reply:
Valid question, this is not immediately obvious:

This module is not part of the public API and was only partially included for the `Manifest` type.
The `Manifest` class no longer exists (it is now a bytes-string)

See the `PyInstaller.depend.utils` entry in `stubtest_allowlist.txt` commented with `# Most modules are not meant to be used, yet are not marked as private`